### DISCUSSION TITLE
What values are initially selected when selecting a hierarchical dimension in the color axis ?

### DISCUSSION THREAD

**Discussion by ortnever** [visualize-admin/visualization-tool#750] [Q&A] [answered]
How is the selection of categories made by default, when selecting a hierarchical dimension (Bar chart, color axis)
It seems to me that all categories at the lowest level are selected by default. That is what I see in the following example:
- Open int.visualize.admin.ch, source lindas: INT
- Search for the dataset "Greenhouse gas emissions by sectors and gases (SAD)", Draft.
- Select the bar chart
- Select  Color
- Select "Source of emission" in the right panel.
- All categories at the lowest level are selected by default.
**--> Is this the default rule? Thanks for the confirmation.**

**Comment by ptbrowne** [comment, marked as answer]
Yes this is the default rule. Initially it was made for maps where @rdataflow wanted to see the smallest (= lowest level) regions first.
